src/Bonsai.ML.Visualizers/MultidimensionalArrayVisualizer.cs

```diff
@@ -0,0 +1,102 @@
+using System;
+using System.Windows.Forms;
+using System.Collections.Generic;
+using System.Reflection;
+using Bonsai;
+using Bonsai.Design;
+using Bonsai.ML.Visualizers;
+using Bonsai.ML.LinearDynamicalSystems;
+using Bonsai.ML.LinearDynamicalSystems.LinearRegression;
+using System.Drawing;
+using System.Reactive;
+using Bonsai.Reactive;
+using Bonsai.Expressions;
+using OxyPlot;
+
+[assembly: TypeVisualizer(typeof(MultidimensionalArrayVisualizer), Target = typeof(double[,]))]
+
+namespace Bonsai.ML.Visualizers
+{
+
+    /// <summary>
+    /// Provides a type visualizer to display the state components of a Kalman Filter kinematics model.
+    /// </summary>
+    public class MultidimensionalArrayVisualizer : DialogTypeVisualizer
+    {
+
+        private int paletteSelectedIndex = 0;
+        private int renderMethodSelectedIndex = 0;
```

Comment:
Looks like these could be removed and leave only the corresponding properties, e.g. with `{ get; set; }`.

Reply:
Updated this as well